src/test/fuzz/FuzzedDataProvider.h

```diff
@@ -0,0 +1,245 @@
+//===- FuzzedDataProvider.h - Utility header for fuzz targets ---*- C++ -* ===//
```

Comment:
Could put this into a separate commit to explain in the commit body where it is taken from exactly (repo+commit)?

Reply:
Good point! Fixed!